bin/deploy

```diff
@@ -20,7 +20,6 @@ while [[ $# > 0 ]]; do
         -u|--user)
         SRC_BRANCH="source"
         DEPLOY_BRANCH="master"
-        shift
```

Comment:
Why was this `shift` deleted? Wouldn't this introduce a bug?

Reply:
Try `bin/deploy --user --src master` to reproduce the issue in the original deploy script.